packages/web-components/fast-ssr/README.md

```diff
@@ -190,6 +190,34 @@ ElementRenderer.disable("my-element");
 templateRenderer.render(html`<my-element></my-element>`);
 ```
 
+### Hydration
+#### `defer-hydration` Attribute
+The `defer-hydration` attribute is an attribute added to each custom-element rendered by the `ElementRenderer`. The presence of the attribute indicates to client-side code that the element should not hydrate it's view. When the attribute is removed, the element is free to hydrate itself.
+
+This attribute is added automatically during custom element rendering to all FAST custom elements. If your app does not require orchestrating element hydration, emission of this attribute can be prevented by configuring the renderer:
+```ts
+const { templateRenderer } = fastSSR({deferHydration: false});
```

Comment:
I'm wondering if we should flip the default on this. Seems like most folks working with the library will want to hydrate automatically, yes?

Reply:
I went back and forth on it. I think *most* applications of a reasonable size will want to defer hydration because they'll have services and contexts that will need to get set up, so hydration order will matter. We can still make it opt-in though, if that is a more predictable interface

Reply:
I think switching the default to be `false` is probably more what's expected, especially if someone is getting started.